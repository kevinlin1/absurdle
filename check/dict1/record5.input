mouse
error
adopt
clean
baker
relay
relax
regal
legal
leggy
