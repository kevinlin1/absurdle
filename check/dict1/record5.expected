2309 words
> mouse
: ⬜⬜⬜⬜🟨

340 words
> error
: 🟨⬜⬜⬜⬜

153 words
> adopt
: ⬜⬜⬜⬜⬜

32 words
> clean
: ⬜🟨🟨⬜⬜

9 words
> baker
: ⬜⬜⬜🟨⬜

3 words
> relay
: ⬜🟩🟨⬜🟩

1 words
> relax
: ⬜🟩🟨⬜⬜

1 words
> regal
: ⬜🟩🟩⬜🟨

1 words
> legal
: 🟩🟩🟩⬜⬜

1 words
> leggy
: 🟩🟩🟩🟩🟩

Absurdle 10/∞

⬜⬜⬜⬜🟨
🟨⬜⬜⬜⬜
⬜⬜⬜⬜⬜
⬜🟨🟨⬜⬜
⬜⬜⬜🟨⬜
⬜🟩🟨⬜🟩
⬜🟩🟨⬜⬜
⬜🟩🟩⬜🟨
🟩🟩🟩⬜⬜
🟩🟩🟩🟩🟩
